scheduler/core/components/optimizer/dummy.py

```diff
@@ -60,9 +61,12 @@ def add(self, group: GroupData, plans: Plans, interval: Optional[Interval] = Non
             if not plan.is_full and plan.site == observation.site:
                 obs_len = plan.time2slots(plan.time_slot_length, observation.exec_time())
                 if plan.time_left() >= obs_len and observation not in plan:
+                    atom_start = 0
+                    atom_end = len(observation.sequence) - 1
```

Comment:
No problem. 😄  I just went over all the code and tried to fix as many warnings as possible.
It was an easy fix... I figured it would take less time than creating a Jira task and iterating on it.